src/Extensions/Logging/InitStaticLogger.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	/// <summary>
+	/// InitStaticLogger is the logger to be used before the uls logger is set
+	/// not to be used as main logger
+	/// </summary>
+	public sealed class InitStaticLogger
```

Comment:
Why is not class static? What do you think about the name `InitializationLogger`, and then a field with logger could be called `Instance`